src/udev/udev-builtin-net_id.c

```diff
@@ -46,6 +46,7 @@
  *   [P<domain>]p<bus>s<slot>[f<function>][u<port>][..][c<config>][i<interface>]
  *                                         — USB port number chain
  *   v<slot>                               - VIO slot number (IBM PowerVM)
+ *   a<vendor>m<model>i<instance>          — ACPI instance id (ARM64 platform devices)
```

Comment:
does this really only apply to ARM64?

I mean, if I look into /sys/devices/platform/ I see a number of ACPI devices there too, on my x86-64 machine. Admittedly they aren't network devices, but indicating that this only matters for ARM64 even though it might in theory apply to all ACPI systems regardless of the architecture sounds misleading to me?